packages/quantic/force-app/main/default/lwc/quanticResultList/quanticResultList.js

```diff
@@ -26,7 +26,11 @@ export default class QuanticResultList extends LightningElement {
    * @param {import("coveo").SearchEngine} engine
    */
   initialize = (engine) => {
-    this.resultList = CoveoHeadless.buildResultList(engine);
+    this.resultList = CoveoHeadless.buildResultList(engine,{
+      options: {
+        fieldsToInclude : ['date', 'filetype'],
```

Comment:
instead of this we should add fieldsToInclude as an @api prop with a default value of `['date', 'filetype']`, It's an option that's missing that we need to add anyway :)